definitions/ext-switch/golden_metrics.yml

```diff
@@ -0,0 +1,27 @@
+cpuUtilization:
+  title: CPU Utilization (%)
+  query:
+    select: average(kentik.snmp.CPU) AS 'CPU Utilization'
+    from: Metric
+    where: "provider = 'kentik-switch'"
```

Comment:
The `where` clauses should not be necessary I think as the GMs will match against telemetry that is already stamped with a GUID.  Is that right @naxhh ?

Reply:
I think that for that to happen you'd have to use [queries](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#roll-up-entities), it might make sense here:

> In the cases that the entity type can be ingested from different sources and you need to provide a different query implementation you can use queries instead of query

Reply:
for context; this is using the same pattern as multiple [AWS Entities](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/definitions/infra-awsalb/golden_metrics.yml) in that `provider` can have multiple values so the `WHERE` seems helpful.

If it's not required because there's an implicit `WHERE entityGuid = '1234abcd'`; then I would agree it's unnecessary.

Reply:
Yeah, when the queries get executed they will be applied to a specific entity, so the `where` would be redundant. But it doesn't break anything anyway.

Reply:
ok; would you like me to remove them @srvaroa? I can get that done real quick so this can get merged.

Reply:
also; would the same apply to the Summary Metrics?

Reply:
Yes! I'll ping @naxhh  to get this approved soon as possible on our end.

Reply:
@thezackm let's leave it for now, we're not 100% sure if that where is important for golden signals.  I can clean it up later if needed.

Reply:
As talking with @srvaroa I have a doubt on a specific use case. So let's keep the provider for now and I'll double check that.